src/components/staking/PageValidator.vue

```diff
@@ -2,6 +2,8 @@
   <TmPage
     :managed="true"
     :data-empty="!validator.operatorAddress"
+    :loading="$apollo.loading"
+    :loaded="!$apollo.loading"
```

Comment:
that doesn't work. this needs to be a value set after the initial load. i.e. in the `result` hook of the `validator` query.

Reply:
Currently it defeats the purpose of `loaded` as it is set to `false` on ever new loading process.

Reply:
It appears to work fine for me. Also when the validator address is unknown.

Reply:
What happens if you delegate to the validator and there is a refetch of the rewards?

Reply:
I see, a new transaction in the subscription would cause it to show the loading component.

Reply:
💯